operations/IT/hardware-buy-back-policy.md

```diff
@@ -0,0 +1,22 @@
+# Hardware Buy Back Policy
+
+Staff members have the option to buy back their existing hardware (laptop, desktop, phone, tablet etc.) either when it gets refreshed for a new one (which is on average, 3 years from purchase date - not necessarily 3 years of employment), or when the staff member is offboarding. 
+
+If the staff member has completed one calendar year or more at Mattermost at the time of offboarding, they can opt to keep their device at no cost. If the staff member hasn't completed one calendar year at the time of offboarding or is receiving an early hardware refresh, they have the option to purchase their hardware from Mattermost at current market value.
+
+If the hardware requires a buyout, IT will send the staff member an email with the determined value. Then if the employee decides to move forward with the purchase, our accounting department will reach out with payment information.
+
+If a staff member decides to retain their hardware, they are required to wipe the device, re-install the base operating system, and remove any and all software and configurations that were supplied by Mattermost. You will be required to sign a document acknowledging you they have not taken or retained any confidential Mattermost software or data.
+
+## Requirements for keeping a device
+
+1. Perform system reset with data deletion:
```

Comment:
should we make a note to the staff member that any and all personal information should be move to a cloud folder before doing the hardware reset?

Reply:
Theoretically we forbid personal usage but I don't think it's a bad idea to make an extra note. Agreed.

Reply:
In my opinion since we forbid personal usage, wouldn't asking them to back up personal data be conflicting messaging?

Reply:
I agree with. I did not know that that we forbid personal data use. There are few changes that need to be made to [How to Spend Company Money Policy](https://handbook.mattermost.com/operations/finance/staff-member-expenses/how-to-spend-company-money) if that is the case under the Mattermost Guidelines:

# 4 - we right now allow staff to use the company laptop for personal reasons. I'll need to update this, which may have push back. When was the policy implemented on personal usage and is there a place in the hanbook that I can reference? 

# 6 - we note that anything over $1,000 is company property. Still true, but I should include the buyback link in this section as well

@danielsischy @DSchalla

Reply:
@DSchalla I'll refer to your opinion here. I don't have any issues with employees using their laptops for personal use within reason as long as it's communicated and outlined as stated in Tim's #4 point. Do we have any security obligations that require us to state that personal use is forbidden? Or has it just been company policy to mitigate risk?